app/src/js/components/Executions/execution-events.js

```diff
@@ -35,8 +35,11 @@ const ExecutionEvents = ({
   const { events } = executionHistory || {};
   const mutableEvents = cloneDeep(events);
   if (mutableEvents) {
-    mutableEvents.forEach((event) => {
+    mutableEvents.forEach((event, index, eventsArray) => {
       event.eventDetails = getEventDetails(event);
+      if (index === 0 || event.name || !event.type.match('LambdaFunction')) return;
+      const prevStep = eventsArray[index - 1];
+      event.name = prevStep.name;
```

Comment:
Since LambdaFunction events do not include a name prop, had to use previous event to get the name.

LambdaFunction events are surrounded by `TaskStateEntered` and `TaskStateExited`, which include the name of the lambda. That's where this is pulling the name from.

Reply:
👍